tests/release/pipelines/tenant_pipelines.go

```diff
@@ -0,0 +1,120 @@
+package pipelines
+
+import (
+	"encoding/json"
+	"fmt"
+
+	tektonutils "github.com/konflux-ci/release-service/tekton/utils"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appservice "github.com/konflux-ci/application-api/api/v1alpha1"
+	"github.com/konflux-ci/e2e-tests/pkg/clients/has"
+	"github.com/konflux-ci/e2e-tests/pkg/constants"
+	"github.com/konflux-ci/e2e-tests/pkg/framework"
+	"github.com/konflux-ci/e2e-tests/pkg/utils"
+	releasecommon "github.com/konflux-ci/e2e-tests/tests/release"
+	releaseApi "github.com/konflux-ci/release-service/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = framework.ReleaseServiceSuiteDescribe("Release service tenant pipeline", Label("release-pipelines", "tenant"), func() {
+	defer GinkgoRecover()
+
+	var fw *framework.Framework
+	AfterEach(framework.ReportFailure(&fw))
+	var err error
+	var compName string
+	var devNamespace string
+	var releasedImagePushRepo = "quay.io/redhat-appstudio-qe/dcmetromap"
+
+	var component *appservice.Component
+	var releaseCR *releaseApi.Release
+
+	BeforeAll(func() {
+		// Initialize the tests controllers
+		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("tenant-dev"))
+		Expect(err).NotTo(HaveOccurred())
+		devNamespace = fw.UserNamespace
+
+		sourceAuthJson := utils.GetEnv("QUAY_TOKEN", "")
+		Expect(sourceAuthJson).ToNot(BeEmpty())
+
+		// Linking the build secret to the pipeline service account in dev namespace.
+		err = fw.AsKubeAdmin.CommonController.LinkSecretToServiceAccount(devNamespace, releasecommon.HacbsReleaseTestsTokenSecret, constants.DefaultPipelineServiceAccount, true)
+		Expect(err).ToNot(HaveOccurred())
+
+		_, err = fw.AsKubeAdmin.HasController.CreateApplication(releasecommon.ApplicationNameDefault, devNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		component = releasecommon.CreateComponent(*fw, devNamespace, releasecommon.ApplicationNameDefault, releasecommon.ComponentName, releasecommon.GitSourceComponentUrl, "", ".", "Dockerfile", constants.DefaultDockerBuildPipelineBundle)
+
+		data, err := json.Marshal(map[string]interface{}{
+			"mapping": map[string]interface{}{
+				"components": []map[string]interface{}{
+					{
+						"component":  compName,
+						"repository": releasedImagePushRepo,
+					},
+				},
+			},
+		})
+		Expect(err).NotTo(HaveOccurred())
+
+		tenantPipeline := &tektonutils.ParameterizedPipeline{}
```

Comment:
I think it would be also nice to have the ServiceAccount and the Timeout set here, to make sure all parameters are taken in consideration.

Reply:
Added "ServiceAccount" and "Timeouts" settings. Can you please review it again?

Reply:
This parameter was renamed to `ServiceAccountName`.

Reply:
Update, thx!